### PR TITLE
Created conference website

### PR DESCRIPTION
**Pull request by Becky449**
Created and added content to the index.html page
Created and added content to the about.html page
added styling for both pages
Using javascript I created a hamburger button for the mobile menu and a dynamic featured speaker section in the homepage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+- NodeJS 
+- Linters 
+## Project Presentation (LOOM)
+Afri-tech Global summit - 23 September 2022 - Watch Video
+
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://becky449.github.io/Capstone/)
```
Comment (whiz25) [inline README.md]: - This link is broken. Please fix it.

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks for all whose support was involved
```
Comment (whiz25) [inline README.md]: - Acknowledge the designer of the mock up. Please add a link to their online profile

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @Becky449, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- Code looks professional
- You used the given design

## Required Changes ♻️
_Check the comments under the review._
- Please add a link to the video presentation for the project. Use loom to record the video.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Becky449  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by the previous reviewer 👍 
- UI design looks real good 🏄 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**